### PR TITLE
Fix two build warning

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

**Comment by embeddedt** [comment]
Thanks!

File: src/lv_objx/lv_img.c [hunk 1]
```diff
@@ -355,8 +355,8 @@ static bool lv_img_design(lv_obj_t * img, const lv_area_t * mask, lv_design_mode
 
         if(ext->cf == LV_IMG_CF_TRUE_COLOR || ext->cf == LV_IMG_CF_RAW) cover = lv_area_is_in(mask, &img->coords);
 
-        const lv_style_t * style = lv_img_get_style(img, LV_IMG_STYLE_MAIN);
```
Comment (kisvegabor) [inline src/lv_objx/lv_img.c:358]: Why was this rename necessary?

Comment (embeddedt) [reply src/lv_objx/lv_img.c:358]: `style` was already declared in the enclosing scope: https://github.com/no1wudi/lvgl/blob/0e2482cdd1cffa53b33b5e87526e5d20daa9e5e0/src/lv_objx/lv_img.c#L349

Comment (kisvegabor) [reply src/lv_objx/lv_img.c:358]: Oh god... I was looking for directly this, but haven't noticed that declaration. 
Thanks!
